packages/components/src/Banner/Banner.test.tsx

```diff
@@ -7,9 +7,11 @@ it("renders a success banner", () => {
   expect(container).toMatchSnapshot();
 });
 
-it("adds an error class to error banners", () => {
```

Comment:
We're no longer using classnames to do any changes to banner , so replaced this test with a test that does the same thing as the other tests. It's still a snapshot, which I know we don't love, but it seemed like a reasonable way to keep a similar level of coverage.